LoopKit/CarbKit/CarbStore.swift

```diff
@@ -945,3 +843,99 @@ extension CarbStore {
         }
     }
 }
+
+extension CarbStore {
+    
+    public struct QueryAnchor: RawRepresentable {
+        
+        public typealias RawValue = [String: Any]
+        
+        public var deletedModificationCounter: Int64
```

Comment:
How exactly would you like me to make them opaque?

Also, I can change the `Int64` to just `Int`.

Reply:
Actually, it is an `Int64` because that matches the type of the field in each Core Data object. I can change the type of the field Core Data Swift object to be `Int`, but the actual data model requires a bit width so there'd have to be some type conversion somewhere along the way.

I will leave it as-is unless you feel strongly about it. If you want it changed, please let me know at what place you'd like the type conversion.

Reply:
By opaque, I mean that `deletedModificationCounter` and `storedModificationCounter` should not be public.

Reply:
Done.